video/avi_decoder.cpp

```diff
@@ -1142,7 +1138,7 @@ AVIDecoder::TrackStatus::TrackStatus() : track(0), chunkSearchOffset(0) {
 AVIDecoder::OldIndex *AVIDecoder::IndexEntries::find(uint index, uint frameNumber) {
 	for (uint idx = 0, frameCtr = 0; idx < size(); ++idx) {
 		if ((*this)[idx].id != ID_REC &&
-				AVIDecoder::getStreamIndex((*this)[idx].id) == index) {
+		    AVIDecoder::getStreamIndex((*this)[idx].id) == index) {
```

Comment:
It was correct and let the developer distinguish the continuation string from the loop body. Needs to be reverted and rule extended if possible.